cpp/oneapi/dal/detail/policy.hpp

```diff
@@ -45,6 +45,11 @@ class ONEDAL_EXPORT host_policy : public base {
 public:
     host_policy();
 
+    static host_policy& get_default() {
```

Comment:
Does `const` work here?

```suggestion
    static const host_policy& get_default() {
```